### ISSUE TITLE
Script for issuing a token

### ISSUE THREAD

**Issue by ziransun** [mozilla-iot/gateway#778]
Create shell script for issuing a token that allow users to experiment with their gateways

**Comment by ziransun** [comment]
I'd like to work on this if it's okay

**Comment by hobinjk** [comment]
Definitely! Thanks for filing the issue. Let me know if you have any questions

**Comment by ziransun** [comment]
I do have a question :).

My initial thought is to create a .js file (e.g. issue_token.js) that requires JSONWebToken module. And then in a .sh file, run "node issue_toke.js". 

In JSONWebToken module, it calls "const ec = require('../ec-crypto');", If I understand correctly the source for "ec-crypto" is a typescript file. From discussions on #179 it seems that we do need webpack or some kind of build/compile process for this.  For a script in /tools, I guess it is an independent code from webpack. Is it right?

Any suggestions?

**Comment by hobinjk** [comment]
Oh no! This will be a bit more difficult than I thought. It looks like https://github.com/TypeStrong/ts-node should work to run the file.

To work around an issue with `ts-node` the exact command to use is `ts-node -O '{"module": "commonjs"}' issue_token.js`.

The shell file could make sure that the `ts-node` command exists and install it otherwise.

**Comment by ziransun** [comment]
Thanks!
